src/System.Net.HttpListener/tests/GetContextHelper.cs

```diff
@@ -59,7 +59,7 @@ public class CustomAsyncResult : IAsyncResult
     public static class Helpers
     {
         public static bool IsWindowsImplementation { get; } = PlatformDetection.IsWindows && PlatformDetection.IsNotOneCoreUAP;
-        public static bool IsNotWindowsImplementation => !IsWindowsImplementation;
+        public static bool IsNotWindowsImplementation => !IsWindowsImplementation && PlatformDetection.IsNotOneCoreUAP;
```

Comment:
I changed the implementation of the IsWindowsImplementation property here:
https://github.com/dotnet/corefx/pull/20265/files#diff-27526fb5f9ee65da1ecd3dcbbea2c85aR61
to actually detect whether it's running against the httpapi.dll based implementation.  The purpose of this property that @hughbe added was really to check whether or not the managed implementation was being used, in particular to make it easier to actually run with the managed implementation on Windows, and have tests respond accordingly.